.circleci/config.yml

```diff
@@ -106,6 +106,18 @@ commands:
       hses_data_password:
         description: "Password used to access the HSES grants and recipient data"
         type: env_var_name
+      smtp_host:
```

Comment:
do we need to merge main into `main-ar-redesign`? I wouldn't expect to see these email parameters as changes from this PR